### ISSUE TITLE
Support custom coverage threshold for new files

### ISSUE THREAD

**Issue by yuhaoju** [adRise/jest-cov-reporter#28]
Have we considered supporting custom coverage threshold for new files? Currently it's either 100% or disabled, for internal projects like CMSUI, it's not always necessary to require 100% coverage for new files. But disabling it means there won't be a check for coverage at all. So I'm wondering if we can support setting a custom threshold just for new files.

**Comment by xueyan-lu** [comment]
Yes, I think we can add it if we want to set customly. But I am not sure if we can just use comments to ignore the lines we don't want to cover.

**Comment by yuhaoju** [comment]
Yes, but that's still some extra work to figure out where to put those comments and test if they work. Sometimes we have to add quite a few comments which makes the code harder to read. A global custom threshold works by default.

**Comment by yuhaoju** [comment]
Thank you!

**Comment by zhiyelee** [comment]
where are we now for this request? @islyn @arunshan

**Comment by xueyan-lu** [comment]
Apologies for the delay in addressing [this ticket](https://app.shortcut.com/tubi/story/529482/support-custom-coverage-threshold-for-new-files) here. I haven't had the opportunity to follow up on it yet. I can attend to it once I've completed a few higher-priority tasks currently on my plate. If someone else is available to assist in the meantime, that would be grateful.
